### ISSUE TITLE
Add more collaborators on npm 

### ISSUE THREAD

**Issue by twolfson** [slack-for-linux/slack-for-linux#56]
The turnaround time from opening a PR to getting it reviewed is frustrating as a contributor and user. Can we add more owners on npm to stop blocking landing critical PRs (e.g. #52)? If we still would like to push everything after code review, can we add more GitHub contributors? 

/cc @wlaurence

**Comment by twolfson** [comment]
It's now been 2 weeks since the initial opening of #49. This is not tolerable for a PR that is preventing people from installing.

@wlaurance If you are reading these issues and have no time to take action, then that is fine but you must communicate that to us (e.g. say "I am busy for the next 2 weeks, I will land this PR within 1 week after that.").

Since I haven't heard anything since the inital opening of #49, I need to assume that the silence will last approximately 21 days (longest stretch for a standalone PR):

https://github.com/wlaurance/slack-for-linux/pull/29

I am going to make an ultimatum for that 21 day mark (June 06 11:59PM PST). My thoughts on options for now:

- Publish pending PRs as `slack-for-linux-beta` but continue to contribute to `slack-for-linux`
- Fork entirely and submit PR with notice of fork for README and place comments in issues about using other fork with this issue as a reference

I am going to ping @madmuffin1 to this thread as he is quite responsive and still waiting on #54 landing to be able to use the multi-team upgrades without pointing to my PR commits.

@madmuffin1 Sorry for picking you but I have to pick someone =/ Do you think these are reasonable terms and a conclusion to come to?

**Comment by madmuffin1** [comment]
@twolfson I totally understand what you are saying, I have been opening slack-for-linux several times in the last week and became a little sad as it looked like another cool tool that was about to die because the original maintainer dissappeared.

If @wlaurance comes back, I would suggest moving this project into a GitHub group and adding the two of you as maintainers. This project had about 1000 downloads from npmjs.com in the last month and is totally worth being further maintained! you guys did an absolutely amazing job with this project! :clap: 

In other words I find your ultimatum absolutely reasonable - but I would wait another 3-5 days for @wlaurance to get back to us. If he doesn't - go ahead and continue your work on fork, preferably in a GitHub group and not your personal repository and also make sure to add someone else you think is trustworthy to act as a co-maintainer.

**Comment by wlaurance** [comment]
Hi everyone. I apologize for the delay. In order to prevent the slowdown, let's add more contributors to the GH project and more owners to the NPM package.

I have been busy and dealing with some personal issues. Never the less, this should have been communicated.

I am adding a time slot to my calendar for Mondays 9am PST to 11am PST to focus on open source (pretty much only this project at the moment).

I will be taking time today to merge PRs and make releases.

@twolfson and @madmuffin1 as you both are more involved, who would you like to see as GH contributors?

**Comment by twolfson** [comment]
@wlaurance Thank you for getting back to us and setting aside time =)

With respect to GitHub contributors, I don't have anyone off the top of my head. I have seen a few approaches to this:

- Add a "Call for contributors" in the README and open an issue for people to apply on
    - Sounds great in practice but doesn't guarantee good contributors
- Add "help wanted" labels to issues to encourage non-core contributions
    - I think we should do this regardless
- Automatically add anyone who submits a PR as a contributor
    - I always find this one great but scary in concept as a contributor can push to `master` at will. In practice, I usually only auto-add contributors that either submit a great PR on the first time (well explained with good coding style) or have a good conversation (quick replies and willing to empathize with author)

What are your thoughts with regard to GitHub and npm rights? If someone is a contributor on GitHub, then should they also have publication rights on `npm`? I am leaning towards "no" because publishes cannot be undone easily on `npm` =/

**Comment by wlaurance** [comment]
I think "help wanted" labels will be good.

We can add add contributors that submit great PRs.

As for `npm`, I will add you now. What is the npm username you'd like to use?

I think that contributors should not get npm access by default.

**Comment by twolfson** [comment]
On `npm`, I am `twolfson`:

https://www.npmjs.com/~twolfson

On that note, I think we are more/less good for the initial intent of this issue. What were your thoughts on @madmuffin1's GitHub organization idea? I think it's good for 3+ contributors as organizations would allow the non-owners to add new contributors. I say 3+ because any new contributor should probably be approved by the owner or 2 core contributors.

**Comment by twolfson** [comment]
By the way, I going to go label crazy now that we are agreed on the "help wanted" ones.

**Comment by wlaurance** [comment]
I agree on setting up a GH organization.
This project seems to be moving to that size, so let's handle this now.

What shall the organization's name be? 'slack-for-linux'?

:+1: on the labels!

**Comment by twolfson** [comment]
`slack-for-linux` sounds like a good organization name :+1:

**Comment by wlaurance** [comment]
done

**Comment by wlaurance** [comment]
You are a owner of the `slack-for-linux` package as well.

**Comment by twolfson** [comment]
Looks like everything is all set up. Closing this issue. Thanks again for taking care of this issue =)

**Comment by madmuffin1** [comment]
Thank you @wlaurance and @twolfson!
